lib/models/application.coffee

```diff
@@ -24,6 +24,8 @@ errors = require('resin-errors')
 
 { isId, findCallback, mergePineOptions, notFoundResponse, treatAsMissingApplication } = require('../util')
 
+LOCKED_STATUS_CODE = 423
```

Comment:
Is there anywhere we can put this constant so that it can be shared between files?

Reply:
I guess it creates a global constant, so I removed it from the application file and kept it only in the device file

Reply:
@MoranF coffeescript will simple hoist the variable to the top of the file, not create a global, so this change won't work :(